kotlin/android/app/src/main/java/dev/firezone/android/core/presentation/MainActivity.kt

```diff
@@ -1,19 +1,9 @@
 /* Licensed under Apache 2.0 (C) 2024 Firezone, Inc. */
 package dev.firezone.android.core.presentation
 
-import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 import dagger.hilt.android.AndroidEntryPoint
 import dev.firezone.android.R
 
 @AndroidEntryPoint
-internal class MainActivity : AppCompatActivity(R.layout.activity_main) {
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
```

Comment:
Turns out that overriding a parent function only to call the parent's version of it doesn't quite make sense.